testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -1910,6 +1911,24 @@ public void testCreateTableWithTableComment()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
```

Comment:
"Deny creating tables with column comment if unsupported" looks solid & only slightly related to the Delta.
Let's separate PR for this.